lang/cs/Org.Apache.REEF.Evaluator/Evaluator.cs

```diff
@@ -80,8 +80,14 @@ public static void Main(string[] args)
                 }
                 AppDomain.CurrentDomain.UnhandledException += UnhandledExceptionHandler;
 
-                Evaluator evaluator = TangFactory.GetTang()
-                    .NewInjector(ReadClrBridgeConfiguration(), ReadEvaluatorConfiguration(args[0]))
+                var fullEvaluatorConfiguration = ReadEvaluatorConfiguration(args[0]);
+                var injector = TangFactory.GetTang().NewInjector(fullEvaluatorConfiguration);
+                var serializer = injector.GetInstance<AvroConfigurationSerializer>();
+                var rootEvaluatorConfiguration =
+                    serializer.FromString(injector.GetNamedInstance<EvaluatorConfiguration, string>());
+                var evaluator = injector.ForkInjector(
+                    ReadClrBridgeConfiguration(),
```

Comment:
So for REEF-217, you will simply remove this one and put IDriverConnection config as a provider, right?